HAPI/llc-test-harness-twobox.py

```diff
@@ -61,19 +67,33 @@ def init_ao(uut, slave=False):
         uut.s0.distributor = "sites={} comms=A pad=0 on".format(AOSITES)
         print "DEBUG -- distributor"
 
+    if PWMSITES != "":
+        for pwmsite in PWMSITES.split(','):
+            pwm = "s{}".format(pwmsite)
+            uut.svc[pwm].pwm_clkdiv = "3e8" # 1000 in hex.
```

Comment:
Say it with code, not comments! prefer
uut.svc[pwm].pwm_clkdiv = "%x" % (1000)        # knobs accepts hex only